### DISCUSSION TITLE
Markers & Multiple Point Navigation Route

### DISCUSSION THREAD

**Discussion by trymeouteh** [organicmaps/organicmaps#8636] [Ideas]
The ability to add temporary markers on the map. These are not bookmarks, but you can add as many markers as you like and in a menu, delete these markers. You are not able to export or import markers.

The ability to have multiple point navigation routes. Instead from going from A to B, having the option to go from A to B to C or A to B to C to D to E and etc. The points can be places on the map or you can use markers.

**Comment by hb0nd** [comment]
multiple point navigation routes are already a thing, while building a route press a place you want to stop at, and press "Add Stop"

what would temporary markers be used for?

**Reply by trymeouteh** [reply]
I do not see a "Add Stop" button anywhere when I tap on a place. All I can see for creating routes is tapping on a place on the screen or finding it in the search and tapping Route To or Route From and then your limited to selecting the other end of the route and are unable to add points in the middle of the route.

**Reply by biodranik** [reply]
@trymeouteh stops can be added at any time when a route is built. Select any point on the map (or use the search), and then press "Add stop" at the bottom.

**Reply by trymeouteh** [reply]
Ok I found the add stop option now. This is not an easy to use interface so I did create a new suggestion

https://github.com/orgs/organicmaps/discussions/8674

Nevertheless markers would be a nice feature to have
